Jenkinsfile.yaml

```diff
@@ -1,39 +1,76 @@
-artifactPublishingConfig:
-    host: 192.32.52.12
-    user: user53
-    credentialId: dummyGitRepo
+# same as defined in current version
+agent:
+#  label: my_label
+#  customWorkspace: path_to_workspace
+#  dockerImage: maven:3-alpine
+#  args: -v /tmp:/tmp
+  # so that tools will be a prt of agent only
+  # tools needs to be defined in the tool configuration
+#  tools:
+#    maven : maven_3.0.1
+#    jdk : jdk8
 
-agent: any
+configuration:
+  # Push PR changes to the target branch if the build will be success.
+  # default value will be false
+  pushPrOnSuccess: false
 
-buildResultPaths:
-    - path-1
-    - path-2
+  prApprovers:
+    - username1
+    - username2
+    - username3
 
-testResultPaths:
-    - path-1
-    - path-2
+# Study about the reports type. Urgent
+reports:
+  - location_of_report_1
+  - location_of_report_2
+
+environment:
+  variables:
+    variable_1: value_1
+    variable_2: value_2
+
+  # Credentials contains only two fields. See pipeline file for how it will be used
+  credentials:
+    - credentialId : fileCredentialId
+      variable : FILE
+
+      # Username will be accessed by LOGIN_USR and Password will be accessed by LOGIN_PSW
+    - credentialId : dummyGitRepo
+      variable : LOGIN
 
 stages:
-    - name: First
-      steps:
-        - sh './scripts/hello'
-        - sleep 5
-    - name: Build
-      steps:
-        - stepName: sh
-          parameters:
-            script: './scripts/build'
-    - name: Tests
-      steps:
-        - stepName: sh
-          defaultParameter: ./scripts/hello
-
-archiveArtifacts:
-    - Jenkinsfile.yaml
-    - scripts/hello.sh
-
-publishArtifacts:
-    - from: Jenkinsfile.yaml
-      to: ~/archives
-    - from: scripts/hello.sh
-      to: ~/archives
+  - name: satge1
+    agent: any
+    steps:
+      - sh:
+          script: "scripts/hello"
+      - sh: "scripts/hello"
+      - sleep:
+          time: 2
+#          unit: SECONDS
+      - sleep: 2
+      - junit:
+          testResults: "target/**.xml"
+          allowEmptyResults: true
+          testDataPublishers:
+            - AutomateTestDataPublisher
+            - JunitResultPublisher:
+                urlOverride: "urlOverride"
+#            - JiraTestDataPublisher:
```

Comment:
We talked about this during the meeting, put a comment to explain why this is commented out and open a Jira to track the problem. Put the Jira number in the comment.